docs/whats-new/dotnet-2020-05.md

```diff
@@ -22,7 +22,7 @@ Welcome to what's new in .NET docs for May 2020. This article lists some of the
 
 ### New articles
 
-- [CentOS 8 Package Manager - Install .NET Core](../core/install/linux-package-manager-centos8.md) - Add install instructions for CentOS 8
+- CentOS 8 Package Manager - Install .NET Core - Add install instructions for CentOS 8
 - Fedora 32 Package Manager - Install .NET Core
```

Comment:
These could be updated to the correct landing pages now

- https://docs.microsoft.com/en-us/dotnet/core/install/linux-centos
- https://docs.microsoft.com/en-us/dotnet/core/install/linux-fedora

Reply:
@adegeo They could, but since the point of the article is "what was added in May 2020", would it still be accurate to update the links?

Reply:
*shrug* either way is fine I guess :) if the link remained, it forwards to the distro page.

Reply:
Since this file will be deleted soon anyway, I'm going to leave it as is. Maybe in the future we can make a note that says the file has since been replaced and add a link to the replacement file.